misk-jdbc/src/main/kotlin/misk/database/DockerPostgresServer.kt

```diff
@@ -92,14 +92,16 @@ class DockerPostgresServer(
     val containerName = CONTAINER_NAME
 
     val runningContainer = docker.listContainersCmd()
-        .withNameFilter(listOf(containerName))
-        .withLimit(1)
-        .exec()
-        .firstOrNull()
+      .withNameFilter(listOf(containerName))
+      .withLimit(1)
+      .exec()
+      .firstOrNull()
     if (runningContainer != null) {
       if (runningContainer.state != "running") {
-        logger.info("Existing Postgres named $containerName found in " +
-            "state ${runningContainer.state}, force removing and restarting")
+        logger.info(
```

Comment:
nit: I don’t like mixing formatting changes with behavior changes in the same PR.

Reply:
Agreed - but this has come in because of the linting/formatter changes earlier this year - it forces compliance on check in if the code has changed.  It saved me from needing to completely reformat the code at the time.  The side effect is extra formatting changes in the file the first time it is changed.